packages/design-system/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
-  "extends": "../../node_modules/@talend/scripts-config-typescript/tsconfig.json",
-  "include": ["cypress/*", "src/*", "custom.d.ts"],
+  "extends": "@talend/scripts-config-typescript/tsconfig.json",
```

Comment:
Should this dep be in dev dep in the package so ?

Reply:
I believe you have it as a dep of the monorepo?